tests/components/otbr/test_config_flow.py

```diff
@@ -328,6 +328,57 @@ async def test_hassio_discovery_flow_sky_connect(
     assert config_entry.unique_id == HASSIO_DATA.uuid
 
 
+async def test_hassio_discovery_flow_connect_zbt1(
```

Comment:
This test could be combined with the basically identical test above (using `parametrize`).